Fix Hide Bottom Navigation #129

enizbilgin opened the pull request:
- Added an option to show bottom navigation.		
- Added a sample showing how to hide bottom navigation.

zawadz88 commented:
Hi @enizbilgin,
Thank you for this PR!
It looks good to me, especially thank you for providing a sample as well.
Could you change this PR so that it merges to the `develop` branch instead though?
It would be also nice if there was a `StepperLayout#setShowBottomNavigation(boolean)` method so that the visibility could be changed programmatically as well, e.g. on a subset of steps only.

enizbilgin commented:
Hi @zawadz88,
I updated this PR, added StepperLayout#setShowBottomNavigation(boolean) method.
I changed the pull request merged to the **develop branch**.
